prog/vnet/load_balancer_nexus.rb

```diff
@@ -57,6 +62,7 @@ def before_run
   label def destroy
     decr_destroy
     strand.children.map { _1.destroy }
+    Prog::Vnet::LoadBalancerNexus.dns_zone&.delete_record(record_name: load_balancer.hostname)
```

Comment:
Is there an economical way to tighten this up to only be okay with it being optional in development while keeping it tightly coupled? (as we gain less from two copies of code). I think you could do it with something like

```
if (binding = expr) && expr 
```